wenet/utils/mask.py

```diff
@@ -145,7 +148,7 @@ def add_optional_chunk_mask(xs: torch.Tensor, masks: torch.Tensor,
     return chunk_masks
 
 
-def make_pad_mask(lengths: torch.Tensor) -> torch.Tensor:
+def make_pad_mask(lengths: torch.Tensor, max_len: int = None) -> torch.Tensor:
```

Comment:
why add the parameter here?

Reply:
It is for tensorrt export. Plus, giving pad mask the max actual length it needs to pad a batch will reduce the risk of the lengths tensor contains wrong maximum length.